python/py-ipython/Portfile

```diff
@@ -68,6 +68,13 @@ if {${name} ne ${subport}} {
                             size    5154336
         python.pep517       no
         depends_lib-append  port:py${python.version}-setuptools
+    } elseif {${python.version} eq 38} {
+        version             8.12.1
```

Comment:
It looks like 8.12.2 was the last version that supported Python 3.8.

Reply:
good point - corrected now! "8.13" came earlier in the release notes and I stopped reading ;)